### PR TITLE
Create Time Bound

### PR DESCRIPTION
**Pull request by yaaowusu00**
This throws an alert when the user is gone for 24 hours. If the user presses yes on the alert, the confirm function returns true, and if they press cancel it returns false.  If confirm returns false, then I restart the app from the beginning of the calculator section. I tested this using 3 milliseconds as the bound instead of one day so I was able to see if it worked without waiting so long but we can also change the bound to maybe 5 hours, or some lower number that would be reasonable, so let me what you think would be a good amount of time to check on the user.

### CODE REVIEW

File: fudge-ratio/src/main/webapp/calculator/index.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
   <body>
     <h1>Fudge Ratio Calculator</h1>
     <p>
-      Select a estimated time, a task and click the start button when you're
+      Select an estimated time and a task, then click the start button when you're
```
Comment (SoyOscarRH) [inline fudge-ratio/src/main/webapp/calculator/index.html:15]: Thanks so much for this, #workingOnWriting

File: fudge-ratio/src/main/webapp/code/mainScreen.js [hunk 2]
```diff
@@ -47,7 +48,18 @@ const showMain = () =>
         text.style.fontWeight = 700;
         finish.disabled = false;
         if (!timerStorage) localStorage.setItem(KEYS.TIME_KEY, Date.now());
-      } else {
+        if (localStorage.getItem(KEYS.TIME_KEY)){
```
Comment (SoyOscarRH) [inline fudge-ratio/src/main/webapp/code/mainScreen.js:51]: WOW! Really cool implementation

File: fudge-ratio/src/main/webapp/code/mainScreen.js [hunk 3]
```diff
@@ -37,8 +37,9 @@ const showMain = () =>
     const text = get("timer-display");
     const startStop = get("start-stop");
     const finish = get("finish");
-
+    const oneDay = 1000*60*60*24;
```
Comment (SoyOscarRH) [inline fudge-ratio/src/main/webapp/code/mainScreen.js]: This in another level of nitpicking but:
`` const oneDay = 1000 * 60 * 60 * 24;``

**Review by SoyOscarRH — APPROVED** [review]

File: fudge-ratio/src/main/webapp/code/mainScreen.js [hunk 4]
```diff
@@ -37,8 +37,9 @@ const showMain = () =>
...
     let id = null;
+    let keepGoing = 1;
```
Comment (SoyOscarRH) [inline fudge-ratio/src/main/webapp/code/mainScreen.js]: this is a boolean, right?
so:
``let keepGoing = true;``

**Review by SoyOscarRH — APPROVED** [review]

File: fudge-ratio/src/main/webapp/code/mainScreen.js (hunk 4 above)
Comment (fancast) [reply fudge-ratio/src/main/webapp/code/mainScreen.js]: I was thinking the same thing as Oscar for this one.

**Comment by fancast** [comment]
Looks good to me!

**Review by fancast — APPROVED** [review]

File: fudge-ratio/src/main/webapp/calculator/index.html (hunk 1 above)
Comment (yaaowusu00) [reply fudge-ratio/src/main/webapp/calculator/index.html:15]: I was just being nit-picky!

**Pull request merged**